addons/repair/models/stock_picking.py

```diff
@@ -102,10 +103,15 @@ def get_repair_stock_picking_action_picking_type(self):
 class Picking(models.Model):
     _inherit = 'stock.picking'
 
-    is_repairable = fields.Boolean(related='picking_type_id.is_repairable')
+    is_repairable = fields.Boolean(compute='_compute_is_repairable', store=True, precompute=True)
```

Comment:
precompute is strange here since the field is not required

Reply:
Finally, I've modified the domains searching on that field such that it is no more stored